lib/uplink/objects.go

```diff
@@ -0,0 +1,76 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package uplink
+
+import (
+	"time"
+
+	"storj.io/storj/pkg/storj"
+)
+
+// ObjectMeta represents metadata about a specific Object
+type ObjectMeta struct {
+	Bucket   string
+	Path     storj.Path
+	IsPrefix bool
+
+	Metadata map[string]string
+
+	Created  time.Time
+	Modified time.Time
+	Expires  time.Time
+
+	Size     int64
+	Checksum string
```

Comment:
should Checksum should be `[]byte`?

Reply:
It should be! I just haven't changed it yet 👍 It will be in the next push though